recipe/meta.yaml

```
@@ -1,48 +1,62 @@
 {% set name = "Chameleon" %}
-{% set version = "3.6.2" %}
+{% set version = "4.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9a9e5e068ee323817705a793cc03538677915c3b99c3f8a4ecba42e62d66862b
+  sha256: 910cd729f6f7f38adad132ee51faa4f3ccc8344a6721aac31aa51a31f5781c1b
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<39]
 
 requirements:
   host:
-    - python
     - pip
+    - python
+    - setuptools >=42
+    - wheel
   run:
     - python
-    - setuptools
+    - importlib-metadata # [py<310]
 
 test:
   imports:
     - chameleon
     - chameleon.tests
     - chameleon.utils
     - chameleon.zpt
+  commands:
+    - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - pytest --pyargs chameleon.tests
+  requires:
+    - pip
+    - pytest
 
 about:
   home: https://chameleon.readthedocs.org
-  license: Repoze
-  license_url: http://repoze.org/license.html
-  license_family: BSD
+  # Zope Public License (ZPL) Version 2.1
+  # This license has been certified as open source. It has also
+  # been designated as GPL compatible by the Free Software
+  # Foundation (FSF).
+  # https://repoze.org/license.html
+  license: ZPL
```

```suggestion
  license: BSD-4-Clause AND ZPL-2.1 AND PSF-2.0
```
Based on the content of https://github.com/malthe/chameleon/blob/4.6.0/LICENSE.txt